src/pages/en/index.html

```diff
@@ -3,6 +3,6 @@
 get-app-enabled: true
 homepage: true
 page-title: Open-Source Project Corona-Warn-App
-page-description: The website of open-source project Corona-Warn-App. The Corona-Warn-App is an app that helps trace infection chains of COVID-19 in Germany.
+page-description: The website of the open-source project from the Corona-Warn-App. The Corona-Warn-App is an app that helps trace infection chains of COVID-19 in Germany.
```

Comment:
grammar
"from" => "for"

Reply:
Corrected in https://github.com/corona-warn-app/cwa-website/pull/1332/commits/2ab687f10cc2cc171e62681496d7121a087c8d5a